docker-compose.yaml

```diff
@@ -510,7 +510,7 @@ services:
         volumes:
             - ./other/rabbitmq/enabled-plugins:/etc/rabbitmq/enabled_plugins
     postgres:
-        image: timescale/timescaledb:1.1.0-pg10
+        image: timescale/timescaledb:2.7.0-pg14
```

Comment:
we should note that this change is not bw-compatible and requires DB to be gone before upgrading

Reply:
Yeah how will we inform the users?

Reply:
Slack, the hard part is when to do this properly.